packages/core/src/utils.ts

```diff
@@ -74,28 +74,13 @@ export function getEventType<TEvent extends EventObject = EventObject>(
   event: Event<TEvent>
 ): TEvent['type'] {
   try {
-    return isString(event) || typeof event === 'number'
```

Comment:
According to the types, it can never be a number anyway. ESLint had a warning with this.